src/hocon.erl

```diff
@@ -28,13 +28,44 @@
 main(Args) ->
     hocon_cli:main(Args).
 
+proplists(Map) when is_map(Map) ->
+    proplists(maps:iterator(Map), [], []).
+proplists(Iter, Path, Acc) ->
+    case maps:next(Iter) of
+        {K, M, I} when is_map(M) ->
+            Child = proplists(maps:iterator(M), [atom_to_list(K)| Path], []),
+            proplists(I, Path, lists:append(Child, Acc));
+        {K, [Bin|_More]=L, I} when is_binary(Bin) ->
+            NewList = [binary_to_list(B) || B <- L],
+            ReversedPath = lists:reverse([atom_to_list(K)| Path]),
+            proplists(I, Path, [{ReversedPath, NewList}| Acc]);
+        {K, Bin, I} when is_binary(Bin) ->
+            ReversedPath = lists:reverse([atom_to_list(K)| Path]),
```

Comment:
not sure we can actually use unicode for keys, maybe worth testing.
and all the binary atom conversions should take utf8 in to consideration.